src/renderer/components/map/MapComponent.js

```diff
@@ -9,10 +9,9 @@ const geojsonExtent = require('@mapbox/geojson-extent')
 const moment = require('moment/moment')
 const formatRelativeTime = require('../conversations/formatRelativeTime')
 const MapLayerFactory = require('./MapLayerFactory')
-const { Slider, Button, Collapse } = require('@blueprintjs/core/lib/esm/index')
```

Comment:
Why did you remove this line?

Reply:
I readded it assuming it was by error removed

Reply:
I believe I changed it to the commonjs counterpart in the parcel pr and it slipped in that way whiles moving stuff back into the ts pr.
Good catch. That"s what reviewing is for :+1: